src/webui/prefjson.cpp

```diff
@@ -114,6 +114,7 @@ QByteArray prefjson::getPreferences()
     data["ip_filter_enabled"] = session->isIPFilteringEnabled();
     data["ip_filter_path"] = Utils::Fs::toNativePath(session->IPFilterFile());
     data["ip_filter_trackers"] = session->isTrackerFilteringEnabled();
+    data["banned_ip_list"] = session->getBannedIPList().join("\n");
```

Comment:
Following your logic we should also rename existing methods like  `session::getBannedIPList()` to something like `session::getListOfManuallyBannedIPAdresses()`. It will be fully semantically correct, but I'm not sure that such rename will resolve more problems than create. What other men would  say?